sig-docs/charter.md

```diff
@@ -0,0 +1,76 @@
+# SIG Docs Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and adheres to the Roles and Organization Management specified in [sig-governance].
+
+## Scope
+
+SIG Docs is responsible for Kubernetes documentation.
+
+The responsibility for feature documentation lies with feature developers. SIG Docs sets standards for feature documentation, provides clear paths for docs contribution, and coordinates documentation updates during quarterly releases.
```

Comment:
> The responsibility for feature documentation lies with feature developers.

The responsibility for authoring kubernetes belongs to the owners of the areas being documented.  This includes documentation for features, concepts, components, opinions, processes, etc.

Reply:
> Is there a link to how the coordination is done? The process doesn't need to be in this doc, but linking resources would be useful.

There is; I've added a link to the docs release playbook.